src/minirest_util.erl

```diff
@@ -37,7 +37,7 @@ pmap_exec(CallerPid, Fun, El, Timeout) ->
   ExecResult =
     receive
       {result, Pid, {normal, Result}} -> Result;
-      {result, Pid, {crash, {C, E, ST}}} -> erlang:raise(C, E, ST);
+      {result, Pid, {crash, {C, E, ST}}} -> {error, {El, {C, E, ST}}};
```

Comment:
If `pmap` caller traps exits, then in case of crash it was stuck forever because:
* the caller pid receives `EXIT` as a message
* but it never handles it, because `pmap_exec` process crashes and `timeout` message is not sent, so `pmap_gather` waits forever.